functions.py

```diff
@@ -0,0 +1,90 @@
+from models import *
```

Comment:
Please, let's move this file to domain. As for me we should separate logic into some files like database.py, payments.py, etc